tap-snapshots/test/lib/utils/exit-handler.js.test.cjs

```diff
@@ -53,7 +54,7 @@ timing npm:load:configScope Completed in {TIME}ms
 timing npm:load Completed in {TIME}ms
 verbose stack Error: Unknown error
 verbose cwd {CWD}/prefix
-verbose  Foo 1.0.0
+verbose Foo 1.0.0
```

Comment:
I think this is from an empty string as the first argument to the logger. I think `npmlog` padded the string in that case which I should replicate here.

Reply:
Shoudl we though?

Reply:
I think in some cases it was being used to indent logs to differentiate them visually. I don't think the empty string as the first param is the way to do it. But it would be nice to still support indenting.